web/src/stream_edit.js

```diff
@@ -596,7 +597,13 @@ export function initialize() {
                 sub.stream_id,
             )}']`,
         );
-        stream_settings_components.sub_or_unsub(sub, $stream_row);
+
+        if ($target.hasClass("subscribe-button")) {
+            // Prevents displaying loading spinner in the stream row.
```

Comment:
It would be good for this comment to specify that we're not showing the spinner when the subscribe/unsubscribe button on the right is clicked. Just reading the if condition does not make it clear.